secrets-provider-test-chart/templates/job.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Values.global.namespace }}
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Values.rbac.serviceAccount.name }}
+      containers:
+      - image: {{ .Values.secretsProvider.image }}
+        imagePullPolicy: Never
+        name: {{ .Values.secretsProvider.name }}
+        env:
+        - name: SECRETS_DESTINATION
+          value: {{ .Values.secretsProviderEnvironment.secretsDestination | quote }}
+
+        # TODO, remove requirement in code
+        - name: CONTAINER_MODE
+          value: init
+
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+
+        - name: CONJUR_APPLIANCE_URL
+          value:  {{ required "conjurApplianceUrl is a required configuration" .Values.secretsProviderEnvironment.conjurApplianceUrl | quote }}
+
+        - name: CONJUR_AUTHN_URL
+          value:  {{ required "conjurAuthnUrl is a required configuration"  .Values.secretsProviderEnvironment.conjurAuthnUrl | quote }}
+
+        - name: CONJUR_ACCOUNT
+          value:  {{ required "serviceAccount is a required configuration"  .Values.rbac.serviceAccount.name | quote }}
+
+        - name: CONJUR_SSL_CERTIFICATE
+          valueFrom:
+            configMapKeyRef:
+              name: {{ .Values.secretsProviderEnvironment.conjurSslCertificateName | quote }}
+              key: ssl-certificate
+
+        - name: CONJUR_AUTHN_LOGIN
+          value:  {{ required "conjurAuthnLogin is a required configuration" .Values.secretsProviderEnvironment.conjurAuthnLogin | quote }}
+
+        - name: SUPPORT_ROTATION
+          value: {{ required "supportRotation is a required configuration"  .Values.secretsProviderEnvironment.supportRotation | quote }}
+
+        - name: SUPPORT_MULTIPLE_APPS
+          value: "true"
+
+        - name: K8S_SECRETS
+          value: {{ required "k8sSecrets is a required configuration"  .Values.secretsProviderEnvironment.k8sSecrets }}
+
+        {{- if .Values.secretsProvider.debug }}
+        - name: DEBUG
+          value: "true"
+        {{- end }}
+      restartPolicy: Never
+  backoffLimit: 0
```

Comment:
These two lines are very hard to read, since they are unindented related to the lines before them.
It's not obvious where they belong, better keep them with lines in the same indentation.